src/pages/BiddingPage/BiddingPage.js

```diff
@@ -0,0 +1,462 @@
+
+function BiddingPage() {
+  const [pageMode, setPageMode] = useState('판매');
+  const [buttonMode, setButtonMode] = useState([false, true]);
+  const [modeChange, setModeChange] = useState(true);
+  const [currentId, setCurrentId] = useState(4);
+  const [value, setValue] = useState('');
+  const today = new Date();
+  const thirtyDaysLater = new Date(today.getTime() + 30 * 24 * 60 * 60 * 1000);
+  const [deadLineDate, setDeadLineDate] = useState(thirtyDaysLater);
+
+  function buttonChange(targetId) {
+    if (targetId === 1) {
+      setButtonMode([true, false]);
+      setModeChange(false);
+    } else {
+      setButtonMode([false, true]);
+      setModeChange(true);
+    }
+  }
+
+  function handleInput(e) {
+    const regex = /^[0-9\b]+$/;
+    let inputValue = e.target.value.replace(/,/g, '');
+    inputValue = Number(inputValue);
+    if (inputValue === '' || regex.test(inputValue)) {
+      setValue(inputValue.toLocaleString());
+    }
+  }
+
+  function handleDeadLine(targetId) {
+    setCurrentId(targetId);
+  }
+
+  function handleDate(targetId) {
+    if (targetId === 1) {
+      const oneDayLater = new Date(today.getTime() + 24 * 60 * 60 * 1000);
+      setDeadLineDate(oneDayLater);
+    } else if (targetId === 2) {
+      const threeDaysLater = new Date(
+        today.getTime() + 3 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(threeDaysLater);
+    } else if (targetId === 3) {
+      const sevenDaysLater = new Date(
+        today.getTime() + 7 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(sevenDaysLater);
+    } else if (targetId === 4) {
+      setDeadLineDate(thirtyDaysLater);
+    } else {
+      const sixtyDaysLater = new Date(
+        today.getTime() + 60 * 24 * 60 * 60 * 1000
+      );
+      setDeadLineDate(sixtyDaysLater);
+    }
+  }
+  const year = deadLineDate.getFullYear();
+  const month = String(deadLineDate.getMonth() + 1).padStart(2, '0');
+  const day = String(deadLineDate.getDate()).padStart(2, '0');
+  const formattedDate = `${year}/${month}/${day}`;
+
+  return (
+    <BiddingContainer>
+      <BiddingBox>
+        <ProductContainer>
+          <ProductImg />
+          <ProductInfo>
+            <ModelNumber>1904927</ModelNumber>
+            <EnglishName>Stussy Plush T-Shirt Black</EnglishName>
+            <KoreanName>스투시 플러쉬 티셔츠 블랙</KoreanName>
+            <ProductSize>ONE SIZE</ProductSize>
+          </ProductInfo>
+        </ProductContainer>
+        <ImmediatePrice>
+          <PurchasePrice>
+            <div>즉시 구매가</div>
+            <p>299,000원</p>
+          </PurchasePrice>
+          <SellPrice>
+            <div>즉시 판매가</div>
+            <p>136,000</p>
+          </SellPrice>
+        </ImmediatePrice>
+        <ModeChangeContainer>
+          {(pageMode === '구매' ? PURCHASE_BUTTON : SELL_BUTTON).map(
+            (data, i) => {
+              return (
+                <ModeButton
+                  key={data.id}
+                  buttonMode={buttonMode[i]}
```

Comment:
```suggestion
                  buttonMode={currentBtn === data.id}
```
이렇게 변경하면 아이디 값에 따라서 다른 스타일을 보여줄 수 있겠습니다.